CMakeLists.txt

```diff
@@ -80,6 +80,13 @@ find_package(USD REQUIRED)
 include(cmake/usd.cmake)
 include(${USD_CONFIG_FILE})
 
+if(${MAYA_APP_VERSION} STRLESS "2019")
```

Comment:
Thanks @mattyjams. Good point. See d1434a2

As for the version, I also agree with earlier comment from @HdC-adsk and you.
We recommend using MAYA_API_VERSION in the *.h,*.cpp for back port compatibility since MAYA_APP_VERSION was introduced since Maya 2019.